index.html

```diff
@@ -1447,6 +1447,39 @@ <h3>publicKeyBase58</h3>
 
       </section>
 
+
+      <section id="publicKeyJwk" about="https://w3id.org/security#publicKeyJwk"
+        typeof="rdf:Property">
+        <h3>publicKeyJwk</h3>
+        <p>
+        See <a href="https://www.iana.org/assignments/jose/jose.xhtml">IANA JOSE</a>. 
+        See also <a href="https://tools.ietf.org/html/rfc7517">RFC 7517</a>.
+        </p>
+        <dl>
+          <dt>Status</dt>
+          <dd property="vs:term_status">stable</dd>
+          <dt>Domain</dt>
+          <dd>Key</dd>
+          <dt>Range</dt>
+          <dd>xsd:string</dd>
+        </dl>
+
+        <pre class="example prettyprint language-jsonld">
+{
+  "id": "did:example:123#_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A",
+  "type": "JsonWebKey2020",
```

Comment:
Blocked by discussion about JsonWebKey2020/JwkVerificationKey2020... we need to resolve this to pull in this example.

Reply:
Will update the name, if the name changes... all of the examples in this repo need a refresh.